src/Illuminate/Redis/Connections/PhpRedisConnection.php

```diff
@@ -4,6 +4,9 @@
 
 use Closure;
 
+/**
+ * @mixin \Predis\Client
```

Comment:
So it should be this right?

/**
 * Create a new Predis connection.
 *
 * @param  \Predis\Client  $client
 * @return void
 */